eli5/xgboost.py

```diff
@@ -91,7 +93,9 @@ def explain_prediction_xgboost(
         target_names=None,
         targets=None,
         feature_names=None,
-        vectorized=False):
+        vectorized=False,
+        expand_missing_features=False,
```

Comment:
hm, we don't have feature_re for explain_prediction methods; maybe we should - it could allow to e.g. show only missing features